Libraries/Components/SegmentedControlIOS/SegmentedControlIOS.ios.js

```diff
@@ -10,39 +10,59 @@
 
 'use strict';
 
-const DeprecatedViewPropTypes = require('DeprecatedViewPropTypes');
-const NativeMethodsMixin = require('NativeMethodsMixin');
-const PropTypes = require('prop-types');
 const React = require('React');
-const ReactNative = require('ReactNative');
 const StyleSheet = require('StyleSheet');
 
-const createReactClass = require('create-react-class');
 const requireNativeComponent = require('requireNativeComponent');
 
 import type {ViewProps} from 'ViewPropTypes';
+import type {NativeComponent} from 'ReactNative';
 
-const RCTSegmentedControl = requireNativeComponent('RCTSegmentedControl');
-
-type DefaultProps = {
-  values: $ReadOnlyArray<string>,
-  enabled: boolean,
-};
-
-type Props = $ReadOnly<{|
+type SegmentedControlIOSProps = $ReadOnly<{|
   ...ViewProps,
-  values?: ?$ReadOnlyArray<string>,
+  /**
+   * The labels for the control's segment buttons, in order.
+   */
+  values: $ReadOnlyArray<string>,
+  /**
+   * The index in `props.values` of the segment to be (pre)selected.
+   */
   selectedIndex?: ?number,
+  /**
+   * Callback that is called when the user taps a segment;
+   * passes the segment's value as an argument
+   */
   onValueChange?: ?Function,
+  /**
+   * Callback that is called when the user taps a segment;
+   * passes the event as an argument
+   */
   onChange?: ?Function,
-  enabled?: ?boolean,
+  /**
+   * If false the user won't be able to interact with the control.
+   * Default value is true.
+   */
+  enabled: boolean,
```

Comment:
Unfortunately, due to this being passed through `React.forwardRef`, it's not able to properly detect `defaultProps`. So it needs to be re-made optional

```suggestion
  enabled?: ?boolean,
```